codemirror.vue

```diff
@@ -188,6 +192,15 @@ export default {
         this.editor.scrollTo(scrollInfo.left, scrollInfo.top)
       }
     },
+    readonly () {
+      this.editor.setOption("readOnly", this.readonly)
+      if (this.readonly) {
+        document.getElementsByClassName('CodeMirror')[0].className += ' readonly'
```

Comment:
Why not use `this.$el` here, @mkapiczy ?

Reply:
this.$el returns a textarea instance which is behing the Codemirror element.

Reply:
Then `document.querySelector('.CodeMirror', this.$el)` or possibly `document.querySelector('.CodeMirror', this.$el.parentElement)`.